src/Symfony/Component/Console/Helper/TableCell.php

```diff
@@ -45,6 +45,14 @@ public function __construct($value = '', array $options = array())
         }
 
         $this->options = array_merge($this->options, $options);
+
+        if (0 === (int) $this->options['colspan']) {
+            throw new InvalidArgumentException(sprintf('The value "%s" is not a valid colspan option.', $this->options['colspan']));
```

Comment:
I think it makes sense to utilize some assertion library like `beberlei/assert`. It's probably not concern of this PR, but such checks are not "regular" exceptions. They might have no descriptive messages, because they are very obvious:
```php
Assertion::greaterThan(0, $this->options['colspan']);
Assertion::greaterThan(0, $this->options['rowspan']);
```

They are similar to type hinting checks, no reason to make very descriptive messages every time, these checks are very cheap.